README.rst

```diff
@@ -382,8 +382,8 @@ And then you can build the ``sdist``, and if possible, the ``bdist_wheel`` too::
 
 To make a release of the project on PyPI, assuming you got some distributions in ``dist/``, the most simple usage is::
 
-    twine register dist/*
-    twine upload --skip-existing dist/*
```

Comment:
You are right. I didn't realize that `twine upload` does take multiple arguments.

What about the `register` command? Do we  have to run that for both the wheel and the tar.gz, or only one of them?